php/commands/core.php

```diff
@@ -1150,6 +1164,29 @@ private function get_updates( $assoc_args ) {
 		return array_values( $updates );
 	}
 
+	private function _maybe_remove_unused_files( $version_from, $version_to) {
```

Comment:
Also, I'm not a fan of methods with `maybe` in the name, as it makes the method ambiguous. We should do the version comparison beforehand.